core/SCsub

```diff
@@ -2,9 +2,11 @@
 
 Import("env")
 
+import os
+
 import core_builders
+
 import methods
```

Comment:
Is this done by `isort`? What's the rule applied here?

It makes sense to me to separate stdlib imports from our local ones, but I'd then expect `core_builders` and `methods` to be adjacent, not separated.